RecoVertex/AdaptiveVertexFinder/plugins/TemplatedVertexArbitrator.h

```diff
@@ -54,6 +54,39 @@ class TemplatedVertexArbitrator : public edm::stream::EDProducer<> {
 	typedef std::vector<VTX> Product;
 	TemplatedVertexArbitrator(const edm::ParameterSet &params); 
 
+	static void fillDescriptions(edm::ConfigurationDescriptions & cdesc) {
+	  edm::ParameterSetDescription pdesc;
+	  pdesc.add<edm::InputTag>("beamSpot",edm::InputTag("offlineBeamSpot"));
+	  pdesc.add<edm::InputTag>("primaryVertices",edm::InputTag("offlinePrimaryVertices"));
+          if( std::is_same<VTX,reco::Vertex>::value ) {
+            pdesc.add<edm::InputTag>("tracks",edm::InputTag("generalTracks"));
+            pdesc.add<edm::InputTag>("secondaryVertices",edm::InputTag("vertexMerger"));
+          } else if (  std::is_same<VTX,reco::VertexCompositePtrCandidate>::value ) {
+            pdesc.add<edm::InputTag>("tracks",edm::InputTag("particleFlow"));
+            pdesc.add<edm::InputTag>("secondaryVertices",edm::InputTag("candidateVertexMerger"));
+          } else {
+            pdesc.add<edm::InputTag>("tracks",edm::InputTag("generalTracks"));
+            pdesc.add<edm::InputTag>("secondaryVertices",edm::InputTag("vertexMerger"));
+          }
+	  pdesc.add<double>("dLenFraction",0.3333);
```

Comment:
this parameter was 0.333 before (one extra 3).
Please change it back.

Reply:
OK will fix momentarily.